esi_leap/api/controllers/v1/utils.py

```diff
@@ -139,3 +139,31 @@ def check_offer_lessee(cdict, offer):
         resource_policy_authorize(
             'esi_leap:offer:offer_admin',
             cdict, cdict, 'offer', offer.uuid)
+
+
+def offer_get_dict_with_added_info(offer,
```

Comment:
Does this have to be on multiple lines? I only ask because the function declaration for `lease_get_dict_with_added_info()` is on one line

Reply:
Probably not, but as long as the linter doesn't complain it should be fine!